amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -548,7 +548,16 @@
                    <configuration>
                       <images>
                          <image>
-                            <name>${image.name}:${image.tag}</name>
+                             <name>${image.name}:${image.tag}</name>
+                             <build>
+                                 <buildx>
+                                     <platforms>
+                                         <platform>linux/amd64</platform>
+                                         <platform>linux/arm64</platform>
+                                     </platforms>
+                                 </buildx>
```

Comment:
Ideally, since this change is repeated several times across multiple pom files, it would be nice to have it centralized.

Reply:
Moved to global pom